ci/cloudbuild/builds/generate-libraries.sh

```diff
@@ -44,6 +46,10 @@ if [ -z "${GENERATE_GOLDEN_ONLY}" ]; then
     --generate_discovery_protos=true \
     --config_file="${PROJECT_ROOT}/generator/generator_config.textproto"
 
+  io::log_h2 "Removing obsolete google/cloud/compute/v1/internal/*.proto"
+  find "${PROJECT_ROOT}/google/cloud/compute/v1/internal/" -name '*.proto' -a ! -newer "${newer_tmp_file}" -exec git rm {} \;
```

Comment:
The common idiom in `git` is to `git rm` *everything* before generating the files, then generate the files and then `git add` what is generated.  I am guessing the idiom won't work because here we both parse and regenerate the `*.proto` files to preserve the field numbers?  If I guessed wrong, consider changing the code.